src/examples/regression_test.json

```diff
@@ -1,139 +1,139 @@
 {
   "Voting_": {
-    "digest": "2f0ca1dca22d7af925d31d23f3d4e37686ec1755a9f069063ae3b6300f6b3920",
+    "digest": "3d94f280d3d143b91564d659073b641b21f6e0ca8d8549c4f9f37434ebeea399",
     "methods": {
       "voterRegistration": {
         "rows": 1260,
-        "digest": "5d6fa3b1f0f781fba8894da098827088"
+        "digest": "9f1d08069b07c165fe425b9b1b83d3ba"
       },
       "candidateRegistration": {
         "rows": 1260,
-        "digest": "abf93b4d926a8743bf622da4ee0f88b2"
+        "digest": "6e9e689b9d1bd2eedcc323c842cf8c87"
       },
       "approveRegistrations": {
-        "rows": 1146,
-        "digest": "197ce95e1895f940278034d20ab24274"
+        "rows": 1147,
```

Comment:
I am actually a bit surprised by this change - this change shouldn't impact this as far as I understand, but maybe something on the Mina branch changed how snarky works :thinking:

Reply:
Thats right! Didnt think about that at all